docs/docs/integrations/providers/databricks.md

```diff
@@ -1,87 +1,111 @@
 Databricks
 ==========
 
-The [Databricks](https://www.databricks.com/) Lakehouse Platform unifies data, analytics, and AI on one platform.
+> [Databricks](https://www.databricks.com/) Intelligence Platform is the world's first data intelligence platform powered by generative AI. Infuse AI into every facet of your business.
 
 Databricks embraces the LangChain ecosystem in various ways:
 
-1. Databricks connector for the SQLDatabase Chain: SQLDatabase.from_databricks() provides an easy way to query your data on Databricks through LangChain
-2. Databricks MLflow integrates with LangChain: Tracking and serving LangChain applications with fewer steps
-3. Databricks as an LLM provider: Deploy your fine-tuned LLMs on Databricks via serving endpoints or cluster driver proxy apps, and query it as langchain.llms.Databricks
-4. Databricks Dolly: Databricks open-sourced Dolly which allows for commercial use, and can be accessed through the Hugging Face Hub
+1. 🚀 **Model Serving** - Access state-of-the-art LLMs or your fine-tuned models on [Databricks Model Serving](https://www.databricks.com/product/model-serving), via a highly available and low-latency inference endpoint. LangChain provides LLM (`Databricks`), Chat Model (`ChatDatabricks`), and Embeddings (`DatabricksEmbeddings`) implementations, streamlines the integration of your models hosted on Databricks Model Serving into your LangChain application.
+2. 📃 **Vector Search** - [Databricks Vector Search](https://www.databricks.com/product/machine-learning/vector-search) is a serverless vector database seamlessly integrated in teh Databricks Platform. Using `DatabricksVectorSearch`, you can incorporate the highly scalable and reliable similarity search engine into your LangChain application.
+3. 📊 **MLflow** - [MLflow](https://mlflow.org/) is an open-source platrofm to manage the MLf lifecycle, including experiment management, evaluation, tracing, deployment, and more. [MLflow LangChain Integration](/docs/integrations/providers/mlflow_tracking) streamline the process of developing and operating modern compound ML systems.
+4. 🌐 **SQL Database** - [Databricks SQL](https://www.databricks.com/product/databricks-sql) is integrated with `SQLDatabase` in LangChain, allowing you to access the auto-optimizing best performant data warehouse.
+5. 💡 **Open Models** - Databricks open-sourced [DBRX](https://www.databricks.com/blog/introducing-dbrx-new-state-art-open-llm) model which allows for commercial use, and can be accessed through the [Hugging Face Hub](https://huggingface.co/databricks/dbrx-instruct).
```

Comment:
Since we're hosting a fleet of these now, should we rephrase this to make a list of models that can be used (non-exhaustive) that highlights DBRX along with other popular open-source transformers-compatible models?